marketpulse/base/static/manifest.webapp

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Mozilla Marketpulse",
+    "description": "Web app to help gather local market data",
```

Comment:
We should be more descriptive and add that we gather local market data for fxos.